Readability.js

```diff
@@ -329,10 +329,25 @@ Readability.prototype = {
   },
 
   _setNodeTag: function (node, tag) {
-    // FIXME this doesn't work on anything but JSDOMParser (ie the node's tag
-    // won't actually be set).
-    node.localName = tag.toLowerCase();
-    node.tagName = tag.toUpperCase();
+    this.log("_setNodeTag", node, tag);
+    if (node.__JSDOMParser__) {
```

Comment:
Now we have benchmarks, is performance really bad if we remove this and use the generic behavior? I still think we shouldn't distinguish between parsers within Readability…